src/sections/xcm/XcmPage.tsx

```diff
@@ -75,6 +75,9 @@ export function XcmPage() {
   const rpcUrlList = useActiveRpcUrlList()
 
   const handleSubmit = async (e: CustomEvent<TxInfo>) => {
+    const isSnowbridge = e.detail?.meta?.tags === "Snowbridge"
```

Comment:
Compare to exported enum in xcm-cfg

`import { tags } from "@galacticcouncil/xcm-cfg"`

```suggestion
    const isSnowbridge = e.detail?.meta?.tags === tags.Tag.Snowbridge
```